Server/Controllers/HelperController.cs

```diff
@@ -22,15 +22,15 @@ public HelperController(ApplicationDbContext context)
         }
 
         [HttpGet]
-        public List<string> GetUsers()
+        [Route("{username}")]
+        public bool CheckUser(string username)
         {
-            var users_list = new List<string>();
-            foreach(var data in context.Users)
+            foreach(var user in context.Users)
```

Comment:
This will load all users from database in memory. This must be avoided (loading will take more time and memory is limited). Just use linq to find the minimum you need. That would be something like
`context.Users.FirstOrDefault(u => u.UserName == username)`
In this case Entity Framework will filter not in memory, but in database.

Reply:
var user = context.Users.FirstOrDefault(u => u.UserName == username);
            if (user != null)
                    return true;
            return false;

like this?

Reply:
oh no, this "if" is redundant. Just use `return user != null`